requirements.txt

```diff
@@ -4,8 +4,10 @@ dockerfile-parse >= 0.0.13
 future
 koji
 mock
+openshift-release >= 0.0.2
 pyyaml >= 5.1
 retry
 requests
 requests_kerberos
+ruamel.yaml >= 0.15
```

Comment:
pyyaml wasn't good enough?

Reply:
It is more like a leftover when I exploring more format control over the YAML dumping. Seems it is not required anymore.